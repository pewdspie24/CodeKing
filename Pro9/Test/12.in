10
1003431797261 1007435307075
1001293198897 1001425354770
1001878339144 1004117168427
1002889592707 1003068808247
1001758436292 1004170835792
1001875533252 1003850454412
1002392161903 1005831659710
1002718517402 1005448032766
1000796488609 1003084585558
1003139741767 1005063040406
