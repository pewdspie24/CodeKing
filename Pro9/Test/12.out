31619938
1815563
23627292
1804590
25318962
21004926
30327885
25019965
25086951
18524146
